lua/luasnip/util/table.lua

```diff
@@ -0,0 +1,26 @@
+---@generic T
+---@param values T|T[]|table<T, boolean>
+---@return table<T, boolean>
+local function normalize_search_table(values)
```

Comment:
Some doc here? Something like "Turns list into set, for quickly checking `contains`-property"?

Reply:
Add some docs

Reply:
Beautiful :+1: